beekeeper-metadata-cleanup/src/test/java/com/expediagroup/beekeeper/metadata/cleanup/service/PagingMetadataCleanupServiceTest.java

```diff
@@ -210,7 +210,7 @@ void metadataCleanerException() {
         .extracting("tableName")
         .containsExactly(tables.get(0).getTableName(), tables.get(1).getTableName());
 
-    List<HousekeepingMetadata> result = metadataRepository.findAll();
+    List<HousekeepingMetadata> result = (List<HousekeepingMetadata>) metadataRepository.findAll();
```

Comment:
Why's the casting needed here?

Reply:
Because  `metadataRepository.findAll()` returns type `Iterable<HousekeepingMetadata>` instead of `List<HousekeepingMetadata>`

Reply:
Use the guava method - `Lists.newArrayList(metadataRepository.findAll())`